### ISSUE TITLE
Color scheme of find results in `find-bar` does not sync with Scratch editor color mode setting

### ISSUE THREAD

**Issue by DNin01** [ScratchAddons/ScratchAddons#6988]
### Description

Scratch's optional high contrast color scheme increases the color contrast of text on blocks, making them easier for some people to read. It's a good idea to make sure text is discernible against colored backgrounds. However, our `find-bar` addon uses some low-contrast color combinations in the list of results, and doesn't adapt to whether you are using high contrast colors. Take a look:

https://github.com/ScratchAddons/ScratchAddons/assets/106490990/791546f9-dc9c-4a59-b457-8b99c33ab7dc

However, some colors do change along with your settings in `editor-theme3`, if you have it enabled - the text is a darker color and is black with a colored background while hovered.

![Find results in find bar with high contrast colors configured in the Customizable block colors addon](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/463686ce-6669-4dea-bcaf-b8d28d9f2554)![Hovered-over find results in find bar with high contrast colors configured in the Customizable block colors addon](https://github.com/ScratchAddons/ScratchAddons/assets/106490990/9e3b2268-aa7c-4e02-9ff4-0d1098d65473)

### Steps to reproduce

1. Make sure `editor-theme3` is disabled.
2. Open the Scratch editor. Set the editor's color mode to High Contrast under Settings.
3. Click on the find bar input.

### Demonstration(s) and additional context

See above.

### Environment

Scratch Addons version: https://github.com/ScratchAddons/ScratchAddons/commit/940056d5a87b18fe9a01b185c0382f1119a9640e

**Comment by DNin01** [comment]
Technically, this isn't a bug, just a potential enhancement, but the current behavior is still undesirable.
